bundles/http/Citrix/AdobeSign/metadata.json

```diff
@@ -55,9 +54,9 @@
   }],
   "scriptMetadata" : {
     "originalFileName" : "script.js",
-    "storedFileName" : "31fff7a5-ffa8-4019-9e2f-e477837f92de.js",
-    "checksum" : "7AF834AF68C910861E6969D86041D999",
+    "storedFileName" : "script.js",
```

Comment:
You can't change the storedFileName if you don't change the name of the stored script file.

Reply:
changes done